ansible/roles/test/files/ptftests/wr_arp.py

```diff
@@ -359,3 +384,160 @@ def req_dut(self, cmd):
         reply = self.q_from_dut.get()
         self.log("reply: %s" % reply)
         return reply
+
+    # make sure setUp is done before run this test case
+    def test_wrarp_advance(self):
+        release = self.get_release_version()
+        if release != 'none' and release < '202205':
```

Comment:
If there is a separate case in test_wr_arp.py, to add skip in the conditional mark would be good to save time.

Reply:
Applied in latest diff